gnuplot/plan.sh

```diff
@@ -0,0 +1,51 @@
+pkg_name=gnuplot
+pkg_origin=core
+pkg_version="5.0.6"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('gnuplot')
+pkg_source="https://sourceforge.net/projects/gnuplot/files/gnuplot/5.0.6/gnuplot-5.0.6.tar.gz"
```

Comment:
using `$pkg_name` and `$pkg_version` here will allow for easier version updates later on.